debugtools/DDR_VM/generate.xml

```diff
@@ -563,7 +563,7 @@
 		<if>
 		  	<equals arg1="${product}" arg2="java8" />
 			<then>
-				<copy file="${output.dir}/libs/j9ddr.jar" todir="/bluebird/tools/ddr/" />
+				<copy file="${output.dir}/libs/j9ddr.jar" tofile="/bluebird/tools/ddr/java8_j9ddr.jar" />
```

Comment:
I'd prefer to keep the prefix as j9ddr.
java8 is not unique as there is a jvm.28 version and a jvm.29 version, so how about using the name j9ddr_jvm29.jar